modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/integration/SortAggregateIntegrationTest.java

```diff
@@ -114,21 +127,80 @@ public void mapReduceAggregate() {
         });
     }
 
+    /** */
+    @Test
+    public void correctCollationsOnMapReduceSortAgg() throws InterruptedException {
+        fillCacheTbl1(grid(0).cache("TEST"), ROWS);
+
+        QueryEngine engine = Commons.lookupComponent(grid(0).context(), QueryEngine.class);
```

Comment:
Perhaps it's better to extend AbstractBasicIntegrationTest to get rid of boilerplate code